stable/postgresql/templates/statefulset.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   serviceName: {{ template "postgresql.fullname" . }}-headless
   replicas: 1
   updateStrategy:
-    type: RollingUpdate
+    type: {{ .Values.updateStrategy.type }}
```

Comment:
Nice! I don't know why it was hardcoded

Reply:
My bad. Duplicated field. It is still duplicated in the slave statefulset... Either you can fix it in this PR or I do a separate PR to fix it.

Reply:
@desaintmartin I also fixed it in `statefulset-slaves.yaml`, no need for a 2nd PR.